scylla-cdc/src/stream_generations.rs

```diff
@@ -0,0 +1,498 @@
+use futures::stream::StreamExt;
+use scylla::batch::Consistency;
+use scylla::cql_to_rust::{FromCqlVal, FromCqlValError, FromCqlValError::BadCqlType};
+use scylla::frame::response::result::{CqlValue, Row};
+use scylla::frame::value::{Timestamp, Value, ValueTooBig};
+use scylla::query::Query;
+use scylla::{FromRow, IntoTypedRows, Session};
+use std::error::Error;
+use std::sync::Arc;
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Ord, PartialOrd, FromRow)]
```

Comment:
Is this working? Aren't you missing `use scylla::cql_to_rust::FromRow`?

Reply:
I have `use scylla::FromRow` below.
It certainly works on a new version of the driver for me. 
Maybe *they* re-exported this struct in the new version.

Reply:
To make it work I have to import both of them (and they do it the same way in the docs iirc).

Reply:
Are you sure that you are using correct scylla Version?
In any case it passes worlflow check from github actions on my fork.

Reply:
Right, it was the version, thanks.